### PR TITLE
Testing scalability data collect

### PR DESCRIPTION
**Pull request by MTCam**
Adds scalability batch scripts for lassen, more bigger cases.

### CODE REVIEW

File: y3prediction/prediction.py [hunk 1]
```diff
@@ -3785,11 +3817,12 @@ def my_rhs(t, state):
 
         # precludes a pre-compiled timestepper
         # don't know if we should do this
-        #state = force_evaluation(actx, state)
+        state = force_evaluation(actx, state)
```
Comment (anderson2981) [inline y3prediction/prediction.py:3820]: we don't need want this, I don't think.

File: y3prediction/prediction.py [hunk 2]
```diff
@@ -2024,6 +2034,15 @@ def get_production_rates(cv, temperature):
         restart_av_smu = actx.np.zeros_like(restart_cv.mass)
         restart_av_sbeta = actx.np.zeros_like(restart_cv.mass)
         restart_av_skappa = actx.np.zeros_like(restart_cv.mass)
+        temperature_seed = force_evaluation(actx, temperature_seed)
+        restart_av_smu = force_evaluation(actx, restart_av_smu)
+        restart_av_sbeta = force_evaluation(actx, restart_av_sbeta)
+        restart_av_skappa = force_evaluation(actx, restart_av_skappa)
+
+        restart_fluid_state = create_fluid_state(
```
Comment (anderson2981) [inline y3prediction/prediction.py:2042]: doing this twice in a row for some reason?

File: y3prediction/prediction.py [hunk 3]
```diff
@@ -2024,6 +2034,15 @@ def get_production_rates(cv, temperature):
         restart_av_smu = actx.np.zeros_like(restart_cv.mass)
         restart_av_sbeta = actx.np.zeros_like(restart_cv.mass)
         restart_av_skappa = actx.np.zeros_like(restart_cv.mass)
+        temperature_seed = force_evaluation(actx, temperature_seed)
+        restart_av_smu = force_evaluation(actx, restart_av_smu)
+        restart_av_sbeta = force_evaluation(actx, restart_av_sbeta)
+        restart_av_skappa = force_evaluation(actx, restart_av_skappa)
```
[the remark this answers is not in the document]
Comment (MTCam) [reply y3prediction/prediction.py]: Removed.

File: y3prediction/prediction.py (hunk 2 above)
Comment (MTCam) [reply y3prediction/prediction.py]: Heh, removed.

**Review by anderson2981 — APPROVED** [review]
yes.

**Pull request merged**